RELEASE_NOTES.md

```diff
@@ -2,23 +2,81 @@
 
 While [CHANGELOG.md](./CHANGELOG.md) contains detailed documentation and links to all of the source code changes in a given release, this document is intended to be aimed at a more comprehensible version of the contents of the release from the point of view of users of Refinery.
 
+## Version 2.0.0
+
+This is a major release of Refinery with a new configuration file format, new samplers, and updated runtime behavior. It has several breaking changes, and so it also comes with a conversion tool.
+
+### Configuration File Changes
+
+The configuration and sampler file formats hav been completely redesigned. Key changes:
+
+- The preferred file format is now YAML instead of TOML. All examples and documentation now use YAML format (the old *_complete.toml files now have 1.x in their names).
+- The config file is now organized into sections for clarity.
+- Sampler rules now require default target named `__default__` to specify the default sampling behavior.
+- Many default values have changed to be more useful.
+- Configurations are now fully validated - misspellings, type errors, faulty indentions, and extra values are now detected. Refinery will no longer run if your configurations are invalid.
+- Documentation for configuration is now automatically generated so that it will stay in sync with the source.
```

Comment:
@TylerHelmuth Do you want to mention that there's documentation generation for both local and docs site locations?

Reply:
Updated

Reply:
```suggestion
- Documentation for configuration is now automatically generated so that it will stay in sync with the source. This means that the [Honeycomb docs](https://docs.honeycomb.io/manage-data-volume/refinery/) stay in sync and you can now generate local copies using the [conversion tool](https://github.com/honeycombio/refinery/tree/main/tools/convert).
```

Reply:
Applied.